### ISSUE TITLE
[Questions] Replace with your question

### ISSUE THREAD

**Issue by nabin-shah** [qnblackcat/uYouPlus#1481]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

I keep getting this error while building using method 1:
![Screenshot 2024-06-05 183203](https://github.com/qnblackcat/uYouPlus/assets/78318505/6ba0924f-a4d6-4721-b7c0-103f1d111ada)
![Screenshot 2024-06-05 183233](https://github.com/qnblackcat/uYouPlus/assets/78318505/172a22a4-5cf7-4aac-818d-1828b978369c)

**Comment by xxxkowal** [comment]
i got the same issues :<

**Comment by ComfortableMilk4454** [comment]
same

**Comment by ComfortableMilk4454** [comment]
@nabin-shah @xxxkowal @arichornloverALT for now until a uYouPlus dev fixes this you can use [my built ipa with youtube v19.19.5](https://drive.google.com/file/d/1mPdfLx8UenM9qNcKwv4La-sj91Fzx413/view?usp=drive_link) which released on may 12, 2024. not the latest yt version but it works lol.

**Comment by nabin-shah** [comment]
@ComfortableMilk4454 is it safe ? just being cautious, is it really malware free?

**Comment by ComfortableMilk4454** [comment]
yes

**Comment by DenuxPlays** [comment]
Got the same thing.
I think this is caused by this warning:
![image](https://github.com/qnblackcat/uYouPlus/assets/83671398/e6dfdd1c-4b61-4be2-93f7-0a608bad76d0)

**Comment by DenuxPlays** [comment]
I fixed it.
When running the workflow change `The version of uYou` from 3.0.3 to 3.0.4

**Comment by phuongzzz** [comment]
same issue

**Comment by Grandicek** [comment]
same :/ 🥇

**Comment by leobarrosc** [comment]
same 😥

**Comment by iqbal1992** [comment]
Seems like the error is because of iSponsorBlock

**Comment by juant87** [comment]
bump

**Comment by iqbal1992** [comment]
> bump. also can anyone point to where I can find how to enable notifications for uyouplus ipa or am I asking a wrong question?

I think it should be in the iPhone settings app under youtube(uyouplus) app. I just checked my and turned them on to check if I do get notifications or no. Will Update if that works.

**Comment by ComfortableMilk4454** [comment]
> > bump. also can anyone point to where I can find how to enable notifications for uyouplus ipa or am I asking a wrong question?
> 
> I think it should be in the iPhone settings app under youtube(uyouplus) app. I just checked my and turned them on to check if I do get notifications or no. Will Update if that works.

it won't unless the apple id you sideloaded with is a developer account. apple limitation.

**Comment by MrAbattoir** [comment]
I had nothing but trouble building my own IPA.
I ended up creating a new standard user on the Mac and trying again.
This time the build worked.
Not sure why.
Maybe a permission issue.

**Comment by juant87** [comment]
> > > bump. also can anyone point to where I can find how to enable notifications for uyouplus ipa or am I asking a wrong question?
> > 
> > 
> > I think it should be in the iPhone settings app under youtube(uyouplus) app. I just checked my and turned them on to check if I do get notifications or no. Will Update if that works.
> 
> it won't unless the apple id you sideloaded with is a developer account. apple limitation.

i do have a paid developer account. i don’t know how to enable them :)

**Comment by The-Jani** [comment]
Currently there's an issue with [iSponsorBlock](https://github.com/Galactic-Dev/iSponsorBlock) most likely caused by the update of [YTHeaders](https://github.com/PoomSmart/YouTubeHeader). Please be patient while @PoomSmart checks it out, and if needed, fixes the issue.

In the meanwhile, you could revert back to commit 8742a0a (or fork [my reverted repo](https://github.com/The-Jani/uYouPlus/)) and build that with 3.0.4.
However until a fix is deployed, some features won't work above YT v19.22.3+.

The home screen bug is probably another uYou bug.

**Comment by nabin-shah** [comment]
@The-Jani in your reverted repo, does youtube homescreen blacks out? I have to turn of the ad if not homescreen blacks out. I guess the places where it blacks out are the place for ad.

**Comment by The-Jani** [comment]
> @The-Jani in your reverted repo, does youtube homescreen blacks out? I have to turn of the ad if not homescreen blacks out. I guess the places where it blacks out are the place for ad.

Yes. Unfortunately, it does on the newest 19.22.6 YT version. I believe that it's an issue in uYou itself, as it was weeks ago.

**Comment by ComfortableMilk4454** [comment]
> > @The-Jani in your reverted repo, does youtube homescreen blacks out? I have to turn of the ad if not homescreen blacks out. I guess the places where it blacks out are the place for ad.
> 
> Yes. Unfortunately, it does on the newest 19.22.6 YT version. I believe that it's an issue in uYou itself, as it was weeks ago.

yeah any devs here that can fix either of these two issues?
